repos/system_upgrade/common/actors/checkfirstpartitionoffset/libraries/check_first_partition_offset.py

```diff
@@ -0,0 +1,64 @@
+from leapp import reporting
+from leapp.libraries.common import grub
+from leapp.libraries.stdlib import api, CalledProcessError, run
+
+SAFE_OFFSET_kiB = 1024  # 1MiB
+
+
+def is_first_partition_offset_ok(device):
+    try:
+        partition_table = run(['parted', device, 'unit', 'kiB', 'print'], split=True)['stdout']
```

Comment:
I would be for running ``parted`` here with the ``-m`` flag so we have a bit more guarantee on the output format. It does not simplify the parsing by much, but feels a bit safer at least to me.

Reply:
Thank you for your kind advice, however, I switched the patch to use `fdisk` instead of `parted` for the sake of not introducing needless dependencies.